src/skin/adminhtml/default/default/omise/omise-dashboard.js

```diff
@@ -0,0 +1,523 @@
+    jQuery.noConflict();
+    jQuery(document).ready(function(){
+
+        // temporary data
+        var chargeData = null, transferData = null;
+
+        // transform charge data into charge table
+        var setChargeTable = function(i, data){
+            var tr = '<tr>';
+
+            if(data){
+                if(data.is_magento){
+                    tr += '<td>฿ '+data.amount_format+'</td>';
+                    tr += '<td>'+data.id+'</td>';
+                    tr += '<td>'+(data.failure_code?'<span class="error-label">Fail</span>':data.captured?'<span class="success-label">Captured</span>': '<span class="warning-label">Authorized</span>')+'</td>';
+
+                    var td3 = jQuery('<td>').html('-');
```

Comment:
@duzydook Do we still need this line after you change your code to concatenate string style ?